.eslintrc.json

```diff
@@ -9,7 +9,8 @@
     "parser": "@typescript-eslint/parser",
     "parserOptions": {
         "ecmaVersion": 12,
-        "sourceType": "module"
+        "sourceType": "module",
+        "project": ["tsconfig.json", "tsconfig.eslint.json"]
```

Comment:
Rule requires `parserOptions.project` which requires a `tsconfig.json` for each file to be linted https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/parser#user-content-parseroptionsproject:~:text=you%20can%20create%20a%20separate%20tsconfig.eslint.json%20as%20follows

Reply:
Is there really no way to avoid an additional `tsconfig` for this?  After all, the TS rules apply only to the stuff in `src`, no?

Reply:
I confirmed that moving `parser` and `parserOptions` to `overrides` does work, however I slightly prefer consistently using the TS parser, especially because `PR-Trigger` provides JS type hints and specifies `@ts-check`. You're right we only run the TS rules on `*.ts` files, so in the latest commit I flipped the base and override settings: from JS defaults and TS overrides to TS defaults and JS exceptions.

I also updated the `PR-Trigger` `@octokit/webhooks` type hints like in https://github.com/DefinitelyTyped/dt-mergebot/pull/307/files#diff-991eb54df1fe7539477567f31fce13f805b0e1c22206c7ba515303ee8bd5546b

Reply:
Redid the whole thing in TS in #317 -- I think that the types mostly follow what you ran into here.